integration/oasis/stationgraph/node.go

```diff
@@ -33,13 +40,29 @@ func newNode() *node {
 	}
 }
 
+// Function isLocationReachable is used to test whether target node is reachable
 func (n *node) isLocationReachable(distance float64) bool {
-	return (n.arrivalEnergy + n.chargeEnergy) > distance
+	return n.chargeEnergy > distance
+}
+
+// calcChargeTime calculates time effort needed from previous final status to current
+func (n *node) calcChargeTime(prev *node, distance float64, strategy chargingstrategy.ChargingStrategyCreator) float64 {
+	arrivalEnergy := prev.chargeEnergy - distance
+	if arrivalEnergy < 0 {
+		glog.Fatalf("Before updateNode should check isLocationReachable() prev.arrivalEnergy=%#v distance=%#v", prev.arrivalEnergy, distance)
+	}
+	//return strategy.EvaluateCost(arrivalEnergy, chargingstrategy.ChargingStatus{ChargingEnergy: n.chargeEnergy}).Duration
+	return 0.0
```

Comment:
want to return `0.0`?

Reply:
Want to return 
```
//return strategy.EvaluateCost(arrivalEnergy, chargingstrategy.ChargingStatus{ChargingEnergy: n.chargeEnergy}).Duration
```
But need other pull to merge, to keep this atomic pull request work, return `0.0`.

Reply:
Fair enough.